content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.md

```diff
@@ -61,7 +61,7 @@ discuss how you can scale and update your Deployments.
 For your first Deployment, you'll use a hello-node application packaged in a Docker
 container that uses NGINX to echo back all the requests. (If you didn't already try
 creating a hello-node application and deploying it using a container, you can do
-that first by following the instructions from the [Hello Minikube tutorial](/docs/tutorials/hello-minikube/).
+that first by following the instructions from the [Hello Minikube tutorial](/docs/tutorials/hello-minikube/)).
```

Comment:
Good catch; thank you! The only thing is that we need to have a period _inside_ the brackets:
```suggestion
that first by following the instructions from the [Hello Minikube tutorial](/docs/tutorials/hello-minikube/).)
```